app/src/main-process/menu/build-default-menu.ts

```diff
@@ -165,6 +167,15 @@ export function buildDefaultMenu({
         accelerator: 'CmdOrCtrl+2',
         click: emit('show-history'),
       },
+      {
+        label: getStashedChangesLabel(isStashedChangesVisible),
```

Comment:
Do you think this could be moved down below the `Go to Summary` command in the list? 

My thinking here is that this group of menu items feels like it's accessed fairly regularly, while stashed changes would be used less frequently, so it doesn't make sense to have it in the middle of this group.